authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -87,22 +100,89 @@ public RangerAuthorizationPlugin(String catalogProvider, Map<String, String> con
     RangerHelper.check(password != null, "Ranger password is required");
     RangerHelper.check(rangerServiceName != null, "Ranger service name is required");
     rangerClient = new RangerClientExtend(rangerUrl, authType, rangerAdminName, password);
-    rangerHelper = new RangerHelper(this, catalogProvider);
+
+    // Initialize and use the configuration to update the privilegesMapping and ownerPrivileges
+    initializeOwnerPrivilegesConfig();
+    initializePrivilegesMappingConfig();
+    policyResourceDefines = initializePolicyResourceDefinesConfig();
+    AuthorizationConfig authorizationConfig = AuthorizationConfig.loadConfig(catalogProvider);
+    overrideAuthorizationConfig(authorizationConfig);
+
+    rangerHelper =
+        new RangerHelper(
+            rangerClient,
+            rangerAdminName,
+            rangerServiceName,
+            privilegesMapping,
+            ownerPrivileges,
+            policyResourceDefines);
+  }
+
+  /** override the default ranger authorization configuration. <br> */
+  protected void overrideAuthorizationConfig(AuthorizationConfig authorizationConfig) {
+    // Update mapping Gravitino privilege name to the Ranger privileges.
+    try {
+      authorizationConfig
+          .getConfigsWithPrefix(AuthorizationConfig.PRIVILEGE_MAPPING_PREFIX)
+          .forEach(
+              (key, value) -> {
+                Privilege.Name gravitinoPrivilege =
+                    Privilege.Name.valueOf(key.trim().toUpperCase());
```

Comment:
hi @jerryshao 
These codes can convert any String to a Gravitino Privilege enum value.